torch/csrc/api/include/torch/mps.h

```diff
@@ -17,5 +26,15 @@ void TORCH_API manual_seed(uint64_t seed);
 /// Waits for all streams on a MPS device to complete.
 void TORCH_API synchronize();
 
+/// Submits the command buffer to run on the GPU
```

Comment:
I feel like these functions would need quite a bit more of documentation?

Also I'm very commit() doesn't take a command buffer or dispatch queue

Reply:
>I feel like these functions would need quite a bit more of documentation?

I can add links to Metal documentation.

>Also I'm very commit() doesn't take a command buffer or dispatch queue
At this time, there's only one active/current command buffer we have in MPSStream.
Also the dispatch_queue_t in this context is just a Mutex, and has nothing to do with GPU streams.

Reply:
> Also I'm very commit() doesn't take a command buffer or dispatch queue

Alban, commit() method works on the Current command buffer object which you can get from `get_command_buffer`. This is the commandBuffer which MPS backend is currently using and encoding work into. This is already using a commitAndContinue feature which allows for concurrency with CPU. Proving handle to the same CommandBuffer to the external users, provide them the same treatment we give as the rest of the op implementation. 
With this context in-mind we decided not to have the commandBuffer passed to commit was to avoid the issue of a user created Command buffer being passed to the commit() method. 
I want to point out, a user can get the device and the Command Queue from the currentCommandBuffer. This will allow them to create new Command buffers if they wanted to. We would like to avoid this mis-match of the CommandBuffers on our APIs.

Reply:
Updated the PR.